.github/workflows/applicationTesting.yml

```diff
@@ -18,9 +18,9 @@ jobs:
       postgres:
         image: postgres:10.8
         env:
-          POSTGRES_USER: ${{ secrets.POSTGRES_DB_USER }}
-          POSTGRES_PASSWORD: ${{ secrets.POSTGRES_DB_PASS }}
-          POSTGRES_DB: ${{ secrets.POSTGRES_DB_NAME }}
+          POSTGRES_USER: ${{ secrets.WORKFLOW_POSTGRES_DB_USER }}
+          POSTGRES_PASSWORD: ${{ secrets.WORKFLOW_POSTGRES_DB_PASS }}
```

Comment:
Why did you decide to use this naming convention? 
For consistency let us keep the name like this 👍🏿 

      ```
POSTGRES_HOST: ${{ secrets.POSTGRES_HOST }}
      POSTGRES_DATABASE: ${{ secrets.POSTGRES_DATABASE_TEST }}
      POSTGRES_USER: ${{ secrets.POSTGRES_USER }}
      POSTGRES_PASSWORD: ${{ secrets.POSTGRES_PASSWORD }}
```